app/resources/admin.html.template

```diff
@@ -899,6 +899,36 @@
   </fieldset>
 
   <fieldset>
+    <legend>Zero-rating</legend>
+
+    <div class="config">
+      <p>Sometimes mobile carriers "zero-rate" our site so that users can see our site without normal data access fee. It is often applied per domain name. But when users access Person Finder, it also loads resource from other domains e.g., Google Maps, Google Analytics, Google Translate by default.</p>
+
+      <p>If you enable zero-rating mode, it disables all those features which loads resource from external domains, so that the users from the carriers can access Person Finder completely for free.</p>
+
+      <p>CAVEAT: Even if you enable zero-rating mode, the photos attached to person records or notes may be loaded from external domains, if the author has specified a URL of the photo instead of uploading it.</p>
```

Comment:
Do you want to file a feature request for routing such images through personfinder domain?